.gitignore

```diff
@@ -8,3 +8,9 @@
 /.gdb_history
 /include/dwarfs/version.h
 /src/dwarfs/version.cpp
+
+# VS build environment
+.vs/
+out/
+CMakeSettings.json
+fbthrift/
```

Comment:
Why was this added?

Reply:
.vs/**,   out/**, CMakeSettings.json  are configuration files and build artefacts generated by Visual Studio. This was added to support this build environment but it is not that important, of course.

fbthrift was added because there are some file generated in-source during build.